{
  "parameters": {
    "serviceName": "apimService1",
    "resourceGroupName": "rg1",
    "api-version": "2019-12-01-preview",
    "subscriptionId": "subid"
  },
  "responses": {
    "200": {
      "body": [
        {
          "location": "West Europe",
          "networkStatus": {
            "dnsServers": [
              "100.78.90.70"
            ],
            "connectivityStatus": [
              {
                "name": "apimgmtsttgzqxxxxxcrsc.blob.core.windows.net",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:43.6997938Z",
                "lastStatusChange": "2019-01-24T21:47:53.851895Z"
              },
              {
                "name": "apimgmtsttgzqxxxxxcrsc.file.core.windows.net",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:44.0439771Z",
                "lastStatusChange": "2019-05-02T21:22:50.6835366Z"
              },
              {
                "name": "apimgmtsttgzqxxxxxcrsc.queue.core.windows.net",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:44.8564232Z",
                "lastStatusChange": "2019-05-02T20:53:26.6260794Z"
              },
              {
                "name": "apimgmtsttgzqxxxxxcrsc.table.core.windows.net",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:44.0439771Z",
                "lastStatusChange": "2019-05-02T20:53:21.6729263Z"
              },
              {
                "name": "ftmrcxxx.database.windows.net",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:44.1845483Z",
                "lastStatusChange": "2019-05-02T21:02:00.9466899Z"
              },
              {
                "name": "https://prod3.metrics.nsatc.net:1886/RecoveryService",
                "status": "failure",
                "error": "Failed to connect to https://prod3.metrics.nsatc.net:1886/RecoveryService with error Unable to connect to the remote server",
                "lastUpdated": "2019-12-18T12:04:30.1214083Z",
                "lastStatusChange": "2019-01-24T22:00:04.2450462Z"
              },
              {
                "name": "https://westeurope.prod.warm.ingestion.msftcloudes.com",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:43.8251399Z",
                "lastStatusChange": "2019-12-18T11:14:43.3860126Z"
              },
              {
                "name": "LocalGatewayRedis",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:50.6376685Z",
                "lastStatusChange": "2019-12-18T11:14:44.4966742Z"
              },
              {
                "name": "prod.warmpath.msftcloudes.com",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:44.4345454Z",
                "lastStatusChange": "2019-05-08T00:09:58.031766Z"
              },
              {
                "name": "Scm",
                "status": "success",
                "error": "",
                "lastUpdated": "2019-12-18T12:04:58.4815899Z",
                "lastStatusChange": "2019-12-18T11:24:48.4795085Z"
              },
              {
                "name": "smtpi-db3.msn.com:25028",
                "status": "failure",
                "error": "Connection smtpi-db3.msn.com:25028 failed with Error Unable to connect to the remote server",
                "lastUpdated": "2019-12-18T12:04:30.1526431Z",
                "lastStatusChange": "2019-12-18T11:23:05.7709057Z"
              }
            ]
          }
        }
      ]
    }
  }
}
